### DISCUSSION TITLE
PixArt + Dreambooth

### DISCUSSION THREAD

**Discussion by wen020** [PixArt-alpha/PixArt-alpha#22] [Q&A] [answered]
I want to know how PixArt + Dreambooth trains a character and how to set the prompt words during inference. Thank you

**Comment by lawrence-cj** [comment]
The training pipeline is exactly the same as Dreambooth. BTW, We may release the PixArt-α + Dreambooth training code in the future. please stay tuned.

**Comment by yujincheng08** [comment, marked as answer]
The diffuser version should work.

Combining the following:

https://huggingface.co/docs/diffusers/v0.22.1/en/api/pipelines/pixart#pixart

https://huggingface.co/docs/diffusers/v0.22.1/en/training/dreambooth#dreambooth
